### PR TITLE
Task/71 create service to connect frontend and backend

### PR DESCRIPTION
**Pull request by mllrdev-jerry-bayoneta**

### CODE REVIEW

File: src/core/services/order.service.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { client } from "@/core/services/client.service"
+import IPurchaseOrder from "@/core/interfaces/purchase-order.interface"
+import IOrderDetail from "@/core/interfaces/order-detail.interface";
+
+const getAllOrder = async () => {
```
Comment (mllrdev-carlo-medel) [inline src/core/services/order.service.ts]:
```suggestion
const getAll = async () => {
```

File: src/core/services/order.service.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      const orders: Array<IPurchaseOrder> = response.data;
+      return orders
+    })
+    .catch( error => {
+      console.log(error)
+      return null
+    })
+}
+
+const getUserOrder = async () => {
```
Comment (mllrdev-carlo-medel) [inline src/core/services/order.service.ts]:
```suggestion
const getAllByUser= async () => {
```

File: src/core/services/order.service.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      const orders: Array<IPurchaseOrder> = response.data;
+      return orders
+    })
+    .catch( error =>{
+      console.log(error)
+      return null
+    })
+}
+
+const createOrder = async (orderDetails: IOrderDetail) => {
```
Comment (mllrdev-carlo-medel) [inline src/core/services/order.service.ts]:
```suggestion
const create = async (orderDetails: IOrderDetail) => {
```

**Review by mllrdev-carlo-medel — APPROVED** [review]

File: src/core/services/order.service.ts (hunk 1 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/services/order.service.ts]: done

File: src/core/services/order.service.ts (hunk 2 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/services/order.service.ts]: done

File: src/core/services/order.service.ts (hunk 3 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/services/order.service.ts]: done

**Pull request merged**